src/main/webapp/i18n/de/exercise.json

```diff
@@ -189,7 +189,8 @@
                     "results": "Die bestehenden Ergebnisse müssen neu berechnet werden!",
                     "confirmtext": "Möchtest du die Änderungen wirklich übernehmen?"
                 }
-            }
+            },
+            "isUsedForTutorial": "Soll es vom Tutor benotet werden"
```

Comment:
I would translate it this way: Tutoren sollen diese Beispiellösung bewerten

Reply:
done